capture-sdk/sdk/src/main/java/net/gini/android/capture/onboarding/DefaultPages.kt

```diff
@@ -15,7 +15,7 @@ sealed class DefaultPages(val onboardingPage: OnboardingPage) {
             R.string.gc_onboarding_align_corners_message,
             ImageOnboardingIllustrationAdapter(
```

Comment:
[Request] Also the return assistant onboarding illustration's content description needs to be changed. Look for `digitalInvoiceOnboardingIllustrationAdapter` in `GiniBank.kt`.